Altis_Life.Altis/core/functions/fn_markers.sqf

```diff
@@ -0,0 +1,45 @@
+/*
+    File: fn_markers.sqf
+    Author: Fresqo
+    Description: sorts markers like house raids and something else i cant remember right now
+    Usage (RETURNS):
+    
+        ARRAY
+        0 - NUMBER (for switch if required)
+        1 - ARRAY (info to use for marker)
+        
+        SECOND ARRAY RETURNS
+        0 - OBJECT/PLAYER (object to be marked etc) DO NOT USE PLAYER UNLESS MODIFYING DUE TO ADMIN MARKER REMOVALS ETC
+        1 - NUMBER (time in seconds marker up)
+        2 - STRING (marker type)
+        3 - STRING (txt to display)
+        4 - STRING (Color)
+            
+        TODO
+        - marker style and size etc
+*/
+
+params [
+    ["_switch",0,[0]],
```

Comment:
What is _switch supposed to be used for? It's not used elsewhere in the code at all...